### PR TITLE
Endpoint-registry rework

### PR DESCRIPTION
**Pull request by akiskips**
Updated the previous [PR](https://github.com/SAP/i40-aas/pull/124), merging from canary
Updates:
- /semanticprotocol API
- Swaggerhub docu: [API](https://app.swaggerhub.com/apis/akiskips/AAS-Semantic-Protocol-API/1.0.1#/)
- initializer uses the upsert calls (/admin/) to fill the endpoint registry

**NOTE**: make purge should be executed beforehand

### CODE REVIEW

File: src/docker/endpoint-registry/Dockerfile [hunk 1]
```diff
@@ -24,6 +24,24 @@ RUN cd /pkg/aas-logger \
...
+    && npm install
+
+
+COPY pkg/aas-logger/ /pkg/aas-logger/
+
+RUN cd /pkg/aas-logger \
+    && npm run clean \
+    && npm run build
+
+
```
Comment (etoster) [inline src/docker/endpoint-registry/Dockerfile:44]: Why add this part at all? 
That whole part is already build with yarn in the same image! 
It crashes the docker build.

Comment (kasimrehman) [reply src/docker/endpoint-registry/Dockerfile:44]: This looks like a merge error. the yarn block is the correct one.

**Comment by akiskips** [comment]
> It seems it is not building on Travis.

From the error logs, I think it has to do with the ENTRYPOINT [ "npm", "start" ]
Dockerfile in Endpoint-Registry. Should it be yarn instead of npm maybe?

**Comment by etoster** [comment]
> > It seems it is not building on Travis.
> 
> From the error logs, I think it has to do with the ENTRYPOINT [ "npm", "start" ]
> Dockerfile in Endpoint-Registry. Should it be yarn instead of npm maybe?

No, this is intended. See https://github.com/SAP/i40-aas/pull/122

File: src/docker/endpoint-registry/Dockerfile (hunk 1 above)
Comment (akiskips) [reply src/docker/endpoint-registry/Dockerfile:44]: I will revert the Dockerfile to the canary version. The changes to the file were faulty introduced

**Pull request merged**